jsk_perception/sample/include/play_rosbag_fetch_go_to_kitchen.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+<launch>
+  <arg name="rosbag" />
+  <arg name="manager" default="rosbag_play_nodelet_manager" />
+  <arg name="launch_nodelet_manager" default="true" />
+  <arg name="start_time" default="0" />
+  <arg name="duration_time" default="0" />
+  <arg name="duration" default="--duration $(arg duration_time)" if="$(eval duration_time != 0)" />
+  <arg name="duration" default="" if="$(eval duration_time == 0)" />
+
+  <arg name="RGB_CAMERA_INFO" value="/head_camera/rgb/throttled/camera_info" />
+  <arg name="RGB_IMAGE" value="/head_camera/rgb/throttled/image_rect_color" />
+  <arg name="QUAT_RGB_CAMERA_INFO" value="/head_camera/rgb/quater/throttled/camera_info" />
+  <arg name="QUAT_RGB_IMAGE" value="/head_camera/rgb/quater/throttled/image_rect_color" />
+  <arg name="DEPTH_CAMERA_INFO" value="/head_camera/depth_registered/throttled/camera_info" />
+  <arg name="DEPTH_IMAGE" value="/head_camera/depth_registered/throttled/image_rect" />
+
+  <param name="use_sim_time" value="true" />
+  <param name="robot_description" textfile="$(find fetch_description)/robots/fetch.urdf" />
+
+  <node pkg="nodelet" type="nodelet" name="$(arg manager)" args="manager" output="screen" />
+
+  <!-- decompress -->
+  <node name="rgb_decompress" pkg="image_transport" type="republish"
+        args="compressed in:=$(arg RGB_IMAGE) raw out:=$(arg RGB_IMAGE)" />
+  <node name="depth_decompress" pkg="image_transport" type="republish"
+        args="compressedDepth in:=$(arg DEPTH_IMAGE) raw out:=$(arg DEPTH_IMAGE)" />
+  <node pkg="nodelet" type="nodelet" name="point_cloud_xyzrgb"
+        args="load depth_image_proc/point_cloud_xyzrgb $(arg manager)" output="screen" >
+    <remap from="rgb/camera_info" to="$(arg RGB_CAMERA_INFO)" />
+    <remap from="rgb/image_rect_color" to="$(arg RGB_IMAGE)" />
+    <remap from="depth_registered/image_rect" to="$(arg DEPTH_IMAGE)" />
+    <remap from="depth_registered/points" to="/head_camera/depth_registered/throttled/points" />
+    <rosparam>
+      queue_size: 100
+    </rosparam>
+  </node>
+  <node name="resize_rgb" pkg="nodelet" type="nodelet"
+        args="load image_proc/resize $(arg manager)" output="screen">
+    <remap from="image" to="$(arg RGB_IMAGE)" />
+    <remap from="camera_info" to="$(arg RGB_CAMERA_INFO)" />
+    <remap from="~image" to="$(arg QUAT_RGB_IMAGE)" />
+    <remap from="~camera_info" to="$(arg QUAT_RGB_CAMERA_INFO)" />
+    <rosparam>
+      scale_width: 0.25
+      scale_height: 0.25
+    </rosparam>
+  </node>
+  <node name="resize_cloud" pkg="nodelet" type="nodelet"
+        args="load jsk_pcl/ResizePointsPublisher $(arg manager)" output="screen">
+    <remap from="~input" to="/head_camera/depth_registered/throttled/points" />
+    <remap from="~output" to="/head_camera/depth_registered/quater/throttled/points" />
```

Comment:
points name can be changed from arguments as other topics, too?

Reply:
Yes, correct. I updated.